### PR TITLE
Ignore Emacs backup and temporary files

### PR DESCRIPTION
**Pull request by pierre-rouleau**
### Description

Just get Git to ignore Emacs backup and temporary files. That should be an impact for anything else.

Fixes # .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,8 @@ current_counterexample.eqc
 *.log
 *.log.*
 doc
+# Emacs Backup files
+*~
+# Emacs temporary files
+.#*
```
Comment (robertoaloi) [inline .gitignore:30]: Hi, thanks for this. Shouldn't the `#` be escaped?

**Comment by pierre-rouleau** [comment]
As far as I know if the first character is not # you don't need to escape it.

File: .gitignore (hunk 1 above)
Comment (pierre-rouleau) [reply .gitignore:30]: As far as I know it needs to be deleted if it is the first character.  
That made me think that one is missing:   \#*

**Comment by robertoaloi** [comment]
Thanks!

**Pull request merged**